src/instruction_graph_generator.cc

```diff
@@ -841,30 +874,33 @@ void generator_impl::add_dependency(instruction* const from, instruction* const
 	m_execution_front.erase(to->get_id());
 }
 
+void generator_impl::add_dependencies_on_access_front(
+    instruction* const accessing_instruction, const access_front& front, const instruction_dependency_origin origin_for_read_write_front) //
```

Comment:
This is my way of forcing a line break on clang-format 🥲  I want to have the opening `{` for a function definition on a separate line whenever the function signature has a line break, otherwise the second line of the parameter list has the same indentation as the function body and it's difficult to tell them apart.

Reply:
Fair enough!